awx/connection.go

```diff
@@ -533,3 +534,49 @@ func (c *Connection) indent(data []byte) []byte {
 	}
 	return buffer.Bytes()
 }
+
+var passwordFilterRegex = regexp.MustCompile("(?i:password|token|authorization|key)")
+
+func filterJsonBytes(bytes []byte) []byte {
+	if len(bytes) == 0 {
+		return bytes
+	}
+	var jsonObj interface{}
+	err := json.Unmarshal(bytes, &jsonObj)
+	if err != nil {
+		glog.Infof("Error parsing: %v", err)
+		return []byte{}
+	}
+	jsonObj = filterJsonObject(jsonObj.(map[string]interface{}))
```

Comment:
What if jsonObj is of a different type? (e.g the server returns a list of projects )

Reply:
Thanks @moolitayer, this cast was a leftover from a previous version and is definitely not needed. I also added an extra test to make sure `filterJsonObject` properly supports lists as the root object.